app.js

```diff
@@ -18,4 +18,192 @@ function close() {
+<button type="button" class="orange-button link-btn">See project</button> 
+  `;
+    works.appendChild(proj);
+  }
+});
+
+const body = document.querySelector('body');
+
+const linkButton = document.querySelectorAll('.link-btn');
+
+linkButton.forEach((par, index) => {
+  par.addEventListener('click', () => {
+    const modal = document.createElement('div');
+    const overlay = document.createElement('div');
+    modal.className = 'modal';
+    const width = window.innerWidth;
+    if (width < 768) {
+      modal.innerHTML = `<div class="modal-header">
+    <h2 class="project-title">${projects[index].name[0]}</h2>
+    <div id="modal-close"><img src="media/Icon.png" alt="close"></div>
+    </div>
+    <ul class="btn no-style-deco" id="btn-modal">
+        <li> <button type="button" class="grey-button modal-btn">${projects[index].technologies[0]}</button></li>
+        <li> <button type="button" class="grey-button modal-btn">${projects[index].technologies[1]}</button></li>
+        <li><button type="button" class="grey-button modal-btn">${projects[index].technologies[2]}</button></li>        
+    </ul>
+    <div id="modal-main">
+    <img id="image" src="${projects[index].image[1]}" alt="popup image">
+    <div class="modal-p">
+    <p class="text">${projects[index].description[2]}</p>
+   <div id="mod-btn">
+
+    <a href="#">
+    <button type="button" class="art-btn link-btn">See live<img class="modal-icon" src="media/Icon-see live.png" alt="icon"></button>    
+    </a>
+    <a href="#">
+    <button type="button" class="art-btn link-btn">See source<img class="modal-icon" src="media/Vector.png" alt="icon"></button>
+  </a> 
+    </div>
+    </div>
+    </div>
+ `;
+    } else if (width > 768) {
+      modal.innerHTML = `<div class="modal-header">
+        <h2 class="project-title">${projects[index].name[1]}</h2>
+        <div id="modal-close"><img src="media/Icon.png" alt="close"></div>
+        </div>
+        <ul class="btn no-style-deco" id="btn-modal">
+            <li> <button type="button" class="grey-button modal-btn">${projects[index].technologies[0]}</button></li>
+            <li> <button type="button" class="grey-button modal-btn">${projects[index].technologies[1]}</button></li>
+            <li><button type="button" class="grey-button modal-btn">${projects[index].technologies[2]}</button></li>        
+        </ul>
+        <div id="modal-main">
+        <img id="image" src="${projects[index].image[1]}" alt="popup image">
+        <div class="modal-p">
+        <p class="text">${projects[index].description[1]}</p>
+       <div id="mod-btn">
+       <a href="#">
+         <button type="button" class="art-btn link-btn">See live<img class="modal-icon" src="media/Icon-see live.png" alt="icon"></button>
+       <a>
+       <a href="#">
+         <button type="button" class="art-btn link-btn">See source<img class="modal-icon" src="media/Vector.png" alt="icon"></button>
+       <a>  
+        </div>
+        </div>
+        </div>
```

Comment:
Great work so far. Kindly consider removing the `else if` section of your code as it's not a best practice to repeat yourself. Instead, you can have just the `if` section of your codebase and use `CSS media queries` to style your model to be `responsive` across all screen sizes.